### ISSUE TITLE
Maybe some simple questions about your paper.

### ISSUE THREAD

**Issue by HenryHZY** [VPGTrans/VPGTrans#2]
1.As mentioned in README, "The stage-1 pre-training requires COCO caption and SBU. The stage-2 also requires VG caption and Laion-COCO." But the paper is not clear enough for the training dataset (COCO+SBU). Could you make it more clear?

2.Figure 11 of VL-Vicuna in the paper, the question for the second image is wrong.

3.By the way, I am not very familiar with the evaluation dataset. Is there a risk of dataset leaking between the evaluation dataset and the training dataset (e.g., shared images)? 

Thank you!

**Comment by waxnkw** [comment]
Thanks for the questions.

1. Actually, the experiments in my paper can be categorized in two parts:
      **Exploration experiments:** exploration results, TaS, TaT. The details are in Section 2.2. Specifically, **only SBU and COCO** are used.
      **Scale-up experiments:** **Table-4 and 5's results**. The details are in Section 4.5. Specifically, **COCO and SBU are used for stage-1** (projector warm-up). **COCO, VG, and LAION-COCO are used for stage-2** (overall fine-tuning). To explain, I also try to use COCO, VG, and LAION-COCO for stage-1 but find no significant difference in the results. Just for simplicity, I use COCO and SBU for stage-1.

2. Thanks so much for pointing out the problem! I have updated the [pdf](https://github.com/VPGTrans/VPGTrans/blob/main/vpgtrans.pdf) on this GitHub.  The arxiv version will be updated after a revision.

3. Let me first introduce the datasets' image sources:
    COCO caption: COCO images
    NoCaps: Open Images
    VQAv2: COCO images
    GQA: visual genome (VG) images
    OK-VQA: COCO images
Thus, training with VG and COCO will lead to the leak of the training set images for VQAv2, GQA, and OK-VQA, but no annotations leak. Training with COCO will leak the training annotations for the COCO caption. NoCaps has a **in-domain set**, just a COCO caption-like part. Thus, training on COCO is somewhat like supervised training but not rigorous direct supervision.

Overall speaking, the zero-shot evaluation of BLIP-2 is acceptable. No direct annotations get leaked for VQA tasks.

**Comment by HenryHZY** [comment]
@waxnkw Thank you very much for your quick and detailed reply! Great project! I have no questions:)
